src/runtime_src/core/edge/user/system_linux.cpp

```diff
@@ -86,6 +90,24 @@ get_os_info(boost::property_tree::ptree &pt)
   }
 
   pt.put("glibc", gnu_get_libc_version());
+
+  // The file is a requirement as per latest Linux standards
+  // https://www.freedesktop.org/software/systemd/man/os-release.html
+  std::ifstream ifs("/etc/os-release");
+  if (ifs.good()) {
+    boost::property_tree::ptree opt;
+    boost::property_tree::ini_parser::read_ini(ifs, opt);
+    std::string val = opt.get<std::string>("PRETTY_NAME", "");
+    if (val.length()) {
+      if ((val.front() == '"') && (val.back() == '"')) {
```

Comment:
I wonder what happens if there are some `"` at the beginning but not the end or the opposite...
Why do we have to handle such an ambiguous syntax at the first place?

Reply:
added comment, its to remove '"' form both ends.